src/EventsPanel.tsx

```diff
@@ -343,9 +343,13 @@ const newEventMachine = newEventModel.createMachine({
       on: {
         '*': [
```

Comment:
It's weird that we accept **any** event here but you assert that the event is of type `EVENT.PAYLOAD`. In such a case, shouldn't this be changed to this?
```suggestion
        'EVENT.PAYLOAD': [
```

Reply:
However, by looking at the code - this wouldn't be exactly correct because other events should result in the invalid state. In such a case I would propose to do the whole try/catch thing in the if block within `cond` that would be responsible for that event to make this more clear. Something like:
```js
if (e.type === 'EVENT.PAYLOAD') {
  // do the try/catch thing
} else {
  return false
}
```

Reply:
Or you could do this:
```js
on: {
  'EVENT.PAYLOAD': [{ cond: () => { /* try/catch thing */ }, target: '.valid' }, '.invalid'],
  '*': '.invalid'
}
```

Reply:
This is definitely subject to refactoring but they're not changed in the scope of this PR 🤷‍♂️. Let's fix in a separate PR?

Reply:
This free flow of my mind (😅 ) has been prompted by the fact that this transition is defined for `*` events but it casts the received event to `EVENT.PAYLOAD`, seemingly not caring about other event types that can actually be received here.

This has been introduced in this PR and was not a problem before (because before the event type was not used here). So I think this should be refactored somehow as part of this PR